data/reusables/actions/allow-specific-actions-intro.md

```diff
@@ -6,7 +6,9 @@ When you select the **Allow select actions**, there are additional options that
 
   To restrict access to specific tags or commit SHAs of an action, use the same `<OWNER>/<REPO>@<TAG OR SHA>` syntax used in the workflow to select the action. For example, `actions/javascript-action@v1.0.1` to select a tag or `actions/javascript-action@172239021f7ba04fe7327647b213799853a9eb89` to select a SHA. For more information, see "[Finding and customizing actions](/actions/learn-github-actions/finding-and-customizing-actions#using-release-management-for-your-custom-actions)."
 
-  You can use the `*` wildcard character to match patterns. For example, to allow all actions in organizations that start with `space-org`, you can specify `space-org*/*`. To add all actions in repositories that start with octocat, you can use `*/octocat*@*`. For more information about using the `*` wildcard, see "[Workflow syntax for GitHub Actions](/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet)." 
+  You can use the `*` wildcard character to match patterns. For example, to allow all actions in organizations that start with `space-org`, you can specify `space-org*/*`. To add all actions in repositories that start with octocat, you can use `*/octocat*@*`. For more information about using the `*` wildcard, see "[Workflow syntax for GitHub Actions](/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet)".
+
+  Actions whose repositories are in the same organization or user account (local actions) are automatically allowed. For example, the action `space-org/action` is allowed in `space-org/project` without adding it to the list.
```

Comment:
Local actions also includes any within an enterprise account.

Reply:
Hm, I believe this information is missing in other places? Does the enterprise UI define local actions?

Reply:
[About GitHub Actions permissions for your repository](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/disabling-or-limiting-github-actions-for-a-repository#about-github-actions-permissions-for-your-repository)

> When you enable local actions only, workflows can only run actions located in your repository or organization.

The text is the same in the enterprise server of the article.

The Web UI is more generic and shows the following in my case:

- **Allow local actions only**
  Only actions defined in a repository within Simran-B can be used

Reply:
@Simran-B This markdown reusable is used in a few places for describing this setting (Actions policies can be configured at the repo/org/enterprise levels). 

* [Disabling or limiting GitHub Actions for a repository](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/disabling-or-limiting-github-actions-for-a-repository)
* [Disabling or limiting GitHub Actions for your organization](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/disabling-or-limiting-github-actions-for-your-organization)
* [Enforcing GitHub Actions policies in your enterprise account](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-your-enterprise/enforcing-github-actions-policies-in-your-enterprise-account)

I'll make a commit to your branch that will describe this so that it applies to all those situations. 🚀

Reply:
@Simran-B I noticed a few other things that need to be fixed with this too, so I opened a new PR in https://github.com/github/docs/pull/1112 and added you as a co-author of those changes 🚀 . I'll close out this PR.